extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -185,6 +196,28 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
       productForPdp = shoppingDataPerPage[0];
     }
 
+    //pdp view
```

Comment:
Please make this comment and the following descriptive. 
It's important to mention analytics. 

Also, we always put a space between `//` and the first word of the comment.